### PR TITLE
test/e2e/apps: Fix static check failures

### PR DESCRIPTION
**Pull request by hwdef**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:

/kind cleanup

**What this PR does / why we need it**:

Fix static check failures

**Which issue(s) this PR fixes**:

Ref: #81657

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Review by SataQiu — COMMENTED** [review]
Thanks @hwdef
/approve
for hack

**Comment by bclau** [comment]
/lgtm

straight-forward changes

File: test/e2e/apps/statefulset.go [hunk 1]
```diff
@@ -776,7 +776,6 @@ var _ = SIGDescribe("StatefulSet", func() {
 			ss, err := c.AppsV1().StatefulSets(ns).Create(ss)
 			framework.ExpectNoError(err)
 			e2esset.WaitForRunningAndReady(c, *ss.Spec.Replicas, ss)
-			ss = e2esset.WaitForStatus(c, ss)
```
Comment (soltysh) [inline test/e2e/apps/statefulset.go]: Even though it should not happen that we block here, I'd not remove this. If this starts failing it'll be a signal we removed important part of the controller code which is responsible for updating `ObservedGeneration`.

**Review by soltysh — CHANGES_REQUESTED** [review]
/lgtm cancel

File: test/e2e/apps/statefulset.go (hunk 1 above)
Comment (hwdef) [reply test/e2e/apps/statefulset.go]: how about 
```
e2esset.WaitForStatus(c, ss)
```
just delete `ss = `

**Pull request closed without merging**